packages/material-ui/src/locale/index.js

```diff
@@ -224,7 +224,7 @@ export const deDE = {
     },
     MuiRating: {
       getLabelText: (value) => `${value} ${value !== 1 ? 'Sterne' : 'Stern'}`,
-      emptyLabelText: 'Empty',
+      emptyLabelText: 'Keine Wertung',
```

Comment:
I'm not sure if this is the correct translation. "Empty" is pretty meaningless. Is there more context? How would this be displayed?

Reply:
You can have more context on https://www.w3.org/WAI/tutorials/forms/custom-controls/#a-star-rating, the label goes instead of `0 stars` in the above link.

Reply:
So we should improve `Empty` as well. Seems like "No Rating" would have actual meaning.

Reply:
It depends on how generic we want to be. Sounds fair.